tester.py

```diff
@@ -0,0 +1,38 @@
+import urllib2
```

Comment:
The Python interpreter was unable to find requests on the environment, so I used urllib2 to write the tester, for hitting the API